spec/system/task_spec.rb

```diff
@@ -1,24 +1,21 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { FactoryBot.create(:project) }
```

Comment:
設定ファイルに追記することで省略可能です。全体のコード量を減らせるので積極的に省略しましょう。
```suggestion
  let(:project) { create(:project) }
```

Reply:
790625980d26adbe9a5e5b3cd8d8921112443ca7 で表記を省略しました。